charts/cortex/templates/ingester/ingester-statefulset.yaml

```diff
@@ -141,3 +142,163 @@ spec:
             {{- toYaml . | nindent 12 }}
           {{- end }}
 {{- end -}}
+{{- if .Values.topologyAware.enabled }}
+{{- range $availabilityCell := .Values.topologyAware.availabilityCells }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "cortex-ingester.fullname-zone-aware" dict "Values" $.Values "Release" $.Release "Chart" $.Chart "AvailabilityCell" $availabilityCell }}
+  labels: {{ include "cortex-ingester.labels" $ | nindent 4 }}
+{{ $.Values.topologyAware.zoneLabel | nindent 4 }}: {{ $availabilityCell }}
```

Comment:
Very very minor. Normally the `nindent` is used when you need multiline indentation - here you could just add 4 spaces in front of `{{` and there would be no need for piping through `nindent`